include/sparrow/dynamic_bitset.hpp

```diff
@@ -561,7 +577,11 @@ namespace sparrow
 
     template <std::integral T>
     dynamic_bitset<T>::dynamic_bitset(size_type n, value_type value)
-        : base_type(storage_type(this->compute_block_count(n), value ? ~block_type(0) : 0), n, value ? 0u : n)
+        : base_type(
+              storage_type(this->compute_block_count(n), value ? block_type(~block_type(0)) : 0),
```

Comment:
For consistency?
```suggestion
              storage_type(this->compute_block_count(n), value ? block_type(~block_type(0)) : block_type(0)),
```

Reply:
Fix a warning about a conversion

Reply:
My remark was unclear: based [on this change on l.525](https://github.com/xtensor-stack/sparrow/pull/90/files#diff-eb6ccf28ffe0d12cb1cfb2b3d4c3871ee6430fd1d558b86880a69e62788f7f33R541), I was wondering if this line also should be modified similarly?